### DISCUSSION TITLE
J2CL libraries

### DISCUSSION THREAD

**Discussion by faveoled** [google/j2cl#196] [Q&A]
Are libraries a thing in J2CL? Is there a way to find them? E.g. for parsing, HTTP calls etc.

**Comment by treblereel** [comment]
@faveoled, what kind of libs are you talking about?

**Reply by faveoled** [reply]
basically any (like JARs in Java)

**Reply by treblereel** [reply]
@faveoled yeap, there are set of ported gwt2 modules plus some frameworks like https://github.com/DominoKit, https://github.com/NaluKit and some mine (xml/json/yaml ser/deser, DI and so on) 
What exactly are you looking for?

**Reply by faveoled** [reply]
For example, I need a ZIP archiver and a QR generator. The problem is I don't know how to find them. For JVM, I could use Maven Central. Is there a quick way to find if the library supports J2CL?

**Reply by treblereel** [reply]
@faveoled, you can ask here https://matrix.to/#/#vertispan_j2cl:gitter.im. Disclaimer: it's not an official google's chat
